canonical-web-and-design/ubuntu.com
# Remove p-list-step__bullet

## Done

- Removeed the p-list-step__bullet as it does not appear to be required and it seems to add an extra number on mobile.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: 
    - http://0.0.0.0:8001/ai/install
    - http://0.0.0.0:8001/download/iot/installation-media
    - http://0.0.0.0:8001/download/iot/intel-iei-tank-870
    - http://0.0.0.0:8001/download/iot/intel-joule-desktop
    - http://0.0.0.0:8001/download/iot/intel-joule
    - http://0.0.0.0:8001/download/iot/intel-nuc-desktop
    - http://0.0.0.0:8001/download/iot/intel-nuc
    - http://0.0.0.0:8001/download/iot/kvm
    - http://0.0.0.0:8001/download/iot/orange-pi-zero
    - http://0.0.0.0:8001/download/iot/qualcomm-dragonboard-410c
    - http://0.0.0.0:8001/download/iot/raspberry-pi-2-3-core
    - http://0.0.0.0:8001/download/iot/raspberry-pi-2-3
    - http://0.0.0.0:8001/download/iot/raspberry-pi-compute-module-3
    - http://0.0.0.0:8001/download/iot/samsung-artik-5-10-server
    - http://0.0.0.0:8001/download/iot/samsung-artik-5-10
    - http://0.0.0.0:8001/download/iot/up-squared-iot-grove-server
    - http://0.0.0.0:8001/download/server/provisioning
    - http://0.0.0.0:8001/kubernetes/install
    - http://0.0.0.0:8001/kubernetes/managed
    - http://0.0.0.0:8001/livepatch/index
    - http://0.0.0.0:8001/openstack/install
    - http://0.0.0.0:8001/openstack/managed
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that there are no changes visually

## Issue / Card

Fixes #5387
